index.html

```diff
@@ -0,0 +1,234 @@
+            <li>CSS</li>
+            <li>ERB</li>
+            <li>RSPec</li>
+          </ul>
+          <a href="https://github.com/Spayco/katchy/" class="card-link link" target="_blank"
+            rel="noreferrer noopener">See project</a>
+        </div>
+      </div>
+
+    </div>
+    <div class="about-me">
+      <section class="about-me-section">
+        <h2>About me</h2>
+        <p class="about-me-text">
+          I am a skilled Full-Stack Web Developer who loves making pages with
+          beautiful UI. I have studied at Microverse, and I am experienced in
+          remote work.
+          <br />
+          I've also been a good helping hand to a lot of developers in
+          different communities!
+          <br />
+          if you'd like to get in contact about potential job opportunities,
+          please reach out!
+        </p>
+        <a href="https://docs.google.com/document/d/1lYq0Kb50TujzJzDWGobIchRI17JMnAiZMF9OBQgNU68/edit?usp=sharing"
+          class="link" target="_blank" rel="noreferrer noopener">
+          Get my resume
+        </a>
+        <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+      </section>
+      <hr />
+      <div class="about-me-section stacks">
+        <ul>
+          <li class="languages list-title">
+            Languages
+            <ul>
+              <li>JavaScript</li>
+              <li>TypeScript</li>
+              <li>Ruby</li>
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>Sass</li>
+            </ul>
+          </li>
+        </ul>
+        <ul>
+          <li class="frameworks list-title">
+            Frameworks
+            <ul>
+              <li>React & Redux</li>
+              <li>Material-UI</li>
+              <li>Jest</li>
+              <li>Bootstrap</li>
+              <li>Ruby on Rails</li>
+              <li>RSPec</li>
+            </ul>
+          </li>
+        </ul>
+        <ul>
+          <li class="skills list-title">
+            Skills
+            <ul>
+              <li>GitHub</li>
+              <li>CodePen</li>
+              <li>GitLab</li>
+              <li>Terminal</li>
+              <li>WSL</li>
+              <li>Remote Pair-Programming</li>
+              <li>Teamwork</li>
+              <li>Mentoring</li>
+              <li>Code Reviews</li>
+              <li>Mobile/Responsive Development</li>
+            </ul>
+          </li>
+        </ul>
+      </div>
+    </div>
+    <section class="contact">
+      <div class="contact_wrap">
+        <h1 class="head">Contact Form</h1>
+        <h3 class="bodyText">If you have an application you are interested in developing,
+          a feature that you need built or a project that needs coding. I’d love to help with it. </h3>
+        <form action="https://formspree.io/f/mzbyvzqb" method="post" class="contact_form">
+          <input type="text" required maxlength="30" placeholder="Alexandra Loremipsum" name="name" id="name" class="form_fields">
```

Comment:
- [ ] It is always a good idea to use descriptive placeholders and,  currently, the name input fields placeholder is a bit confusing. I think it's better to use `Full name`, `First name` `last name` or just `name` so a novice user will not be confused.